Makefile

```diff
@@ -13,6 +13,10 @@ export GO15VENDOREXPERIMENT=1
 # doesn't allow +, so we use -.
 VERSION := 0.0.1-$(shell date "+%Y%m%d%H%M%S")
 
+DEV_ENV_IMAGE := quay.io/deis/go-dev:0.1.0
+DEV_ENV_WORK_DIR := /go/src/github.com/deis/${SHORT_NAME}
+DEV_ENV_CMD := docker run --rm -v ${PWD}:${DEV_ENV_WORK_DIR} -w ${DEV_ENV_WORK_DIR} ${DEV_ENV_IMAGE}
```

Comment:
I think that would be a bad idea.  By _not_ mounting the host's `$GOPATH` to the container, we are actually ensuring that the binary is built in a clean room where the only source of dependencies is what's in the `vendor` dir.